.github/meson_actions/action.yml

```diff
@@ -0,0 +1,29 @@
+name: MesonBuildTest
```

Comment:
Why is this in `.github/meson_actions/` rather than `.github/workflows/`?

Reply:
I just followed the same pattern of `.github/actions`

Reply:
This doesn't look like an action but more like a regular CI job, so it should be under `workflows/`. This change in other CI files for (e.g.) the `setup-python` action is something I don't quite understand:
```diff
-    - uses: ./.github/actions
+    - uses: ./.github/meson_actions
```

Reply:
Okay, I see what this does and looked at the history; it's been like this for quite a while. The multiple levels of indirection and then ending up calling `tools/travis-*.sh` scripts makes debugging difficult, and most of it is just cruft by now. But best to clean it up only after we get rid of the `setup.py` build I think.